code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -726,6 +726,31 @@
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/sheet/plastic(location)
 
+/datum/chemical_reaction/glitter
+	name = "glitter"
+	id = /datum/reagent/glitter
+	results = list(/datum/reagent/glitter = 10)
+	required_reagents = list(/datum/reagent/oil = 5, /datum/reagent/toxin/acid = 2, /datum/reagent/ash = 3, /datum/reagent/aluminium = 2)
+	required_temp = 340 //arbitrarily lower than plastic to prevent conflict
+
+/datum/chemical_reaction/glitter/pink
+	name = "glitter"
+	id = /datum/reagent/glitter
```

Comment:
wrong copy pasta.
name and id should be unique.
do the same to other reactions.